tests/console/verify_efi_mok.pm

```diff
@@ -93,13 +94,14 @@ sub check_efi_state {
         record_info "Fallback", "EFI label: $found->{label}";
     }
 
-    # return only if SecureBoot is off or image has booted n Fallback mode
-    return if (get_var('DISABLE_SECUREBOOT', 0) || (!$found->{exec} && !$expected->{exec}));
-
-    my $issuer_regex = qr/Secure\s+Boot\s+CA/;
-    diag("Check presence of signature in shim");
-    assert_script_run("pesign -S -i $expected->{mount}/$found->{exec}");
-    push @errors, 'No openSUSE/SUSE keys found in keyring(/proc/keys)' if script_output('cat /proc/keys') !~ $issuer_regex;
+    if (!get_var('DISABLE_SECUREBOOT', 0) && $found->{exec} && $expected->{exec}) {
```

Comment:
Might need `exists($expected->{exec})`.

Reply:
I haven't noticed any warning regarding using unexistent/undef key.

I think, it should be `undef`, in the worst case https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/c6fbed426ac1018a89f780c600ee3a6e33f2d922/tests/console/verify_efi_mok.pm#L59

Reply:
Can be, I don't really know perl, but just assumed that getting a member of `undef` would at least cause a warning...

Reply:
Seems like perl complains only about expressions, not when `undef` variable is accessed in condition. :man_shrugging: 

```
#!/usr/bin/perl

use strict;
use warnings;

my $a = undef;

print "Printing \$a = $a\n";

unless ($a) {
    print "No warning in condition\n";
}
unless ($b) {
    print "No warning in condition\n";
}
```

Yields:

```
╰─❯ perl warns.pl
Use of uninitialized value $a in concatenation (.) or string at warns.pl line 8.
Printing $a =
No warning in condition
No warning in condition
```

Reply:
I can't say I'm surprised...